src/libs/storage/storage.ts

```diff
@@ -20,14 +19,6 @@ export const getShouldMigrateKeyMetaNullToKeyMetaCreatedAt = (keystoreKeys: Stor
     return internalKeyWithoutMeta || externalKeyWithoutCreatedAt
   })
 
-export const needsCashbackStatusMigration = (
-  cashbackStatusByAccount: Record<Account['addr'], CashbackStatus | LegacyCashbackStatus>
-) => {
-  return Object.values(cashbackStatusByAccount).some(
-    (value) => typeof value === 'object' && value !== null && 'cashbackWasZeroAt' in value
-  )
-}
-
```

Comment:
I agree this is no longer needed. But the bigger picture is that the `cashbackStatusByAccount` in storage is no longer needed. In order to clean up the storage for our existing users - need to write a migration in the StorageConroller that simply cleans it, in case it exists.